docs/_data/table/common-configuration.csv

```diff
@@ -210,12 +209,6 @@ alluxio.zookeeper.leader.connection.error.policy,"SESSION"
 alluxio.zookeeper.leader.inquiry.retry,"10"
 alluxio.zookeeper.leader.path,"/alluxio/leader"
 alluxio.zookeeper.session.timeout,"60s"
-aws.accessKeyId,""
-aws.secretKey,""
```

Comment:
The following is still in `common-configuration.yaml`
```
aws.accessKeyId:
  'The access key of S3 bucket.'
aws.secretKey:
  'The secret key of S3 bucket.'
```

Should they be removed?

Reply:
check against one of the previous release branches? ex. release-2.6.2

Reply:
yeah I think we keep those two